apps/nameguard.io/src/components/organisms/Header.tsx

```diff
@@ -52,7 +52,7 @@ export const Header = () => {
           </div>
 
           <div className="lg:hidden">
-            <IconButton className="!p-2" variant="ghost">
+            <IconButton variant="ghost">
               <NGSearchIcon />
```

Comment:
It looks to me like the size of the search icon button is smaller than the size of the hambuger menu icon button.

Please compare the hover states.

![CleanShot 2024-09-21 at 00 27 41](https://github.com/user-attachments/assets/4e9f5380-10a0-4eaa-b721-3b01ad4a5591)

Additionally, as I reduce the width of the screen, the search field gets too close to the NameGuard logo.

![CleanShot 2024-09-21 at 00 29 19](https://github.com/user-attachments/assets/1f9c76d3-0f1c-48e3-afb0-c7d7192a7665)

Reply:
Fixed